app/routes/index.tsx

```diff
@@ -24,42 +24,28 @@ export const loader: LoaderFunction = async ({ request }) => {
   const endTime = dayjs().add(1, 'months').endOf('month').unix() * 1000;
 
   try {
-    console.log(getUserSelfData(process.env.API_HOST ?? ''));
-
-    const {
-      data: { username },
-    } = await axios.get(getUserSelfData(process.env.API_HOST ?? ''), {
-      headers: {
-        'Content-Type': 'application/json',
-        Cookie: cookie,
-      },
-    });
-    console.log({ username });
-    console.log(getUserCalendarId(process.env.API_HOST ?? '', username));
-
-    const {
-      data: {
-        rcal: { ownerId },
-      },
-    } = await axios.get(getUserCalendarId(process.env.API_HOST ?? '', username), {
-      headers: {
-        'Content-Type': 'application/json',
-        Cookie: cookie,
-      },
-    });
-    console.log({ ownerId });
-    const GET_EVENTS = getEvents(process.env.API_HOST ?? '', ownerId, startTime, endTime);
-
-    const response = await axios.get(GET_EVENTS, {
-      headers: {
-        'Content-Type': 'application/json',
-        Cookie: cookie,
-      },
-    });
-    return json<LoaderData>({ events: response.data.data, ENV: baseUrls, error: null });
+    const {data} = await axios.get(getUserSelfData(process.env.API_HOST ?? ''));
+    if(data?.username) {
+      const {data: selfData } = await axios.get(getUserCalendarId(process.env.API_HOST ?? '', data?.username))
```

Comment:
Don't need to again use optional chaining here, since we already have it on line no 28.

Reply:
fixed